src/System.Windows.Forms.Design/tests/System/Drawing/Design/ToolboxItemCollectionTests.cs

```diff
@@ -7,18 +7,24 @@
  namespace System.Drawing.Design.Tests	
 {	
     public class ToolboxItemCollectionTests	
-    {	
-        private ToolboxItemCollection CreateToolboxItemCollection()	
-        {	
-            ToolboxItem[]  tools = { new ToolboxItem(), new ToolboxItem() };	
-
-             return new ToolboxItemCollection(tools);	
-        }	
-
+    {
         [Fact]
-        public void TestToolboxItemCollection_Throw_NotImplemented_Exception()
+        public void TestToolboxItemCollection_Creation()
```

Comment:
```suggestion
        public void ToolboxItemCollection_Creation()
```
Should be named after the class it tests

Reply:
Ah a lot of these seem to be doing that. Would you mind changing them? Not sure why this was not caught earlier

Reply:
Done